src/nft/components/bag/Bag.tsx

```diff
@@ -336,4 +338,25 @@ const Bag = () => {
   )
 }
 
+const BagContainer = styled.div<{ $renderBelowNavbar: boolean }>`
+  position: fixed;
+  display: flex;
+  flex-direction: column;
+  top: 0px;
+  width: 100%;
+  height: 100%;
+  right: 0px;
+  background: ${({ theme }) => theme.backgroundSurface};
+  border-left-style: solid;
+  border-color: ${({ theme }) => theme.backgroundOutline};
+  border-width: 1px;
+  color: ${({ theme }) => theme.textPrimary};
+  z-index: ${Z_INDEX.dropdown};
```

Comment:
Need extra attention on this one! Previously, with Sprinkles, this Z-INDEX was set to `modalOverTooltip`. That means we would have to create a new `Z_INDEX` value of `1090` to render over a tooltip.

However, I don't think that's expected - it's a bag and I think it should render more or less on the same level as the nav bar. When there's a popup or a tooltip (especially rendered in the bottom right corner), I don't think it should render below the bag. It's potentially dangerous (see screenshots in Jira).

I used a `dropdown` value instead. Please note that this is lower value than z-index of the navbar. Thanks to that, we can actually provide a background for a transparent navbar on the NFT Explore page (see screenshots in Jira for more context).